app/javascript/react/components/Video_game_folder/ReviewFormContainer.js

```diff
@@ -8,6 +10,7 @@ const ReviewFormContainer = props => {
     videogame_id: props.gameId
   });
   const [errors, setErrors] = useState("")
+  const [starsKey, setStarsKey] = useState(Math.random())
```

Comment:
why are we setting this to random? why do we need a stars key?  while there will be multiple ReactStars components on the page, they wont be in an array, so a key shouldnt be needed.

Reply:
Hey Nick! So that state, with the Math.Random()  and the ReactStar key is because the component as is doesn't have a built-in "clear form" functionality. What we did here was the suggested work around found on the github page for this package. Here is the link to the work around comment:  https://github.com/n49/react-stars/issues/68#issuecomment-662561542